src/styles/blocks/stars.css

```diff
@@ -0,0 +1,12 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../../images/Star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). It's recommended to stick to one direction for margins to avoid potential margin collapse issues. Consider applying margin in a single direction, like `margin-top` or `margin-bottom`, instead of `margin-right`.